CHANGELOG.md

```diff
@@ -22,6 +22,14 @@ Instead we recommend checking for the disabled attribute using [`$button.hasAttr
 
 This change was introduced in [pull request #2830: Set the boolean disabled attribute consistently in the button component](https://github.com/alphagov/govuk-frontend/pull/2830).
 
+#### Remove Internet Explorer 8 stylesheets, settings and mixins
```

Comment:
@querkmachine I liked your suggestion on Slack

Would you mind if we removed this CHANGELOG entry in favour of a (future) combined entry for all the IE8, IE9 and IE10 things we've taken out?

Probably can't say we've stopped supporting IE8 until we've removed the internal documentation too